policyengine_us/parameters/gov/states/hi/tax/income/exemptions/disability_exemptions/individual.yaml

```diff
@@ -0,0 +1,13 @@
+description: Hawaii allows this amount of money for one disabled individual (any filing status).
+values:
+  2021-01-01: 7_000
+
+metadata:
+  period: year
+  unit: currency-USD
+  label: "Hawaii exemption amount for disabled individual"
+  reference:
+  - title: 2022 Hawaii Resident Income Tax Instructions #page 20, line 25
```

Comment:
need legal code reference (law document)

Reply:
In the legal code for 2022, the value of base exemption is 1040, while in the 2022 form, the value is 1144, so which value should we use here? The legal code link is as below: Hawaii Income Tax Law, HRS §235-54 §18-235-54-(c).


https://files.hawaii.gov/tax/legal/har/har_235.pdf#page=99